spa-react/src/store/slices/usersSlice.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { IUser } from "src/hooks/use-getUsers.hook";
+export interface IProps {
+  users: IUser[];
+}
+interface IInitialState {
+  users: {
+    users: IUser[];
+  };
+}
+const initialState: IInitialState = {
+  users: {
+    users: [],
```

Comment:
Why do you need to have users in users?

Reply:
Firestore want to get data in object, thats why i have created object with key "users" and have putted array with 100 users there